### PR TITLE
Add Short plugins

### PR DESCRIPTION
**Pull request by jonathantneal**
- Adds [`Shorthand Color`](https://github.com/jonathantneal/postcss-short-color)
- Adds [`Shorthand Font-Size`](https://github.com/jonathantneal/postcss-short-font-size)
- Adds [`Shorthand Position`](https://github.com/jonathantneal/postcss-short-position)
- Adds [`Shorthand Size`](https://github.com/jonathantneal/postcss-short-size)
- Adds [`Shorthand Spacing`](https://github.com/jonathantneal/postcss-short-spacing)
- Adds [`Shorthand Text`](https://github.com/jonathantneal/postcss-short-text)

### CODE REVIEW

**Comment by himynameisdave** [comment]
@jonathantneal sorry that I haven't fixed #18 yet, it would have made this easier. Thanks though :+1:

File: plugins.json [hunk 1]
```diff
@@ -1819,5 +1819,82 @@
       "extensions"
     ],
     "author": "MadLittleMods"
+  },
+  {
+    "name": "Shorthand Spacing",
```
Comment (himynameisdave) [inline plugins.json]: @jonathantneal name should be the repo name such as `postcss-short-spacing`

File: plugins.json [hunk 2]
```diff
@@ -1819,5 +1819,82 @@
...
+      "extensions",
+      "shortcuts"
+    ],
+    "packs": [
+      "postcss-short"
+    ],
+    "author": "jonathantneal"
+  },
+  {
+    "name": "Shorthand Size",
```
Comment (himynameisdave) [inline plugins.json]: @jonathantneal same as above, and for the other ones, just use the repo name (I don't want spaces/capitals in the plugin names)

**Comment by jonathantneal** [comment]
I have corrected the `"name"` issue.

**Comment by himynameisdave** [comment]
@jonathantneal :+1: thanks 10/10 will merge

**Pull request merged**